### PR TITLE
feat: add default type

### PR DESCRIPTION
**Pull request by sumo-drosiek**
Signed-off-by: Dominik Rosiek <drosiek@sumologic.com>

### CODE REVIEW

**Comment by pmalek-sumo** [comment]
Do we expect the tests to pass?

**Comment by sumo-drosiek** [comment]
Rather yes than no :)

File: sumologic_collectd_metrics/metrics_config.py [hunk 1]
```diff
@@ -127,7 +134,7 @@ def parse_config(self, config):
         if ConfigOptions.url not in self.conf:
             raise Exception('Specify %s in collectd.conf.' % ConfigOptions.url)
 
-        if not self.types:
+        if not self.types and not self.default_type:
```
Comment (gizmo-rt) [inline sumologic_collectd_metrics/metrics_config.py:137]:
```suggestion
        if not self.types or not self.default_type:
```

File: sumologic_collectd_metrics/metrics_config.py [hunk 2]
```diff
@@ -146,32 +153,35 @@ def parse_config(self, config):
...
-                        self.collectd.warning('Cannot parse data source %s on type %s'
-                                         % (data_source, type_name))
+            with open(db, 'r') as file:  # pylint: disable=W1514  # due to python2.7
+                for line in file:
+                    fields = line.split()
+                    if len(fields) < 2:
+                        continue
+                    type_name = fields[0]
+                    if type_name[0] == '#':
                         continue
```
Comment (gizmo-rt) [inline sumologic_collectd_metrics/metrics_config.py:182]: Can we merge this under one `if` statement ?

File: sumologic_collectd_metrics/metrics_config.py [hunk 3]
```diff
@@ -146,32 +153,35 @@ def parse_config(self, config):
...
+        values = []
+        for data_source in fields:
+            data_source = data_source.rstrip(',')
+            ds_fields = data_source.split(':')
+
+            if ds_fields is None:
+                self.collectd.warning('Cannot parse data source %s on type %s'
+                                    % (data_source, type_name))
+                continue
+            values.append(ds_fields)
```
Comment (gizmo-rt) [inline sumologic_collectd_metrics/metrics_config.py:167]:
```suggestion
           dirty = []
            if ds_fields is  not None:
                values.append(ds_fields)
            else
                dirty.append(data_source)    
sources = ','.join(map(str, dirty))                
self.collectd.warning('Cannot parse data source %s ', % (sources, type_name))                  
```

**Comment by sumo-drosiek** [comment]
@gizmo-rt Thank you for the review, but this change probably is going to be rejected in favor of #35

**Pull request closed without merging**